### PR TITLE
Box shadow updates

### PR DESCRIPTION
**Pull request by vitthalr**
## Current Behavior
Datepicker, Menu, Popup and dropdown component in Fluent UI has different box shadow/ border.

## New Behavior
Changed box shadow and border to make them consistent wrt the approved design.

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/3a1e2cf7f929612683c82173b227e40fbe4de593'>3a1e2cf7f929612683c82173b227e40fbe4de593</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by ling1726** [comment]
@notandrew @daisygeng I'm not up to date with design changes, would any of you be able to check this is what is intended?

**Comment by daisygeng** [comment]
> @notandrew @daisygeng I'm not up to date with design changes, would any of you be able to check this is what is intended?

Thanks for calling this out! Yes I am aware, it was meant to address a few inconsistencies in the shadows being used across components, goal is still to match the shadow values from Fluent 2. (i.e. in calendar, a dropdown component's shadow looked different from a popup or menu's)

This was the sandbox illustrating the issue: https://jzow1q.csb.app/

File: packages/fluentui/react-northstar/src/themes/teams/components/Menu/menuStyles.ts [hunk 1]
```diff
@@ -40,6 +40,9 @@ export const menuStyles: ComponentSlotStylesPrepared<MenuStylesProps, MenuVariab
           // primary has hardcoded grey border color
           border: `${v.borderWidth} solid ${primary ? v.primaryBorderColor : v.borderColor || colors.border}`,
           borderRadius: siteVariables.borderRadiusMedium,
+          ...(submenu && {
+            borderColor: 'transparent',
```
[the remark this answers is not in the document]
Comment (vitthalr) [reply packages/fluentui/react-northstar/src/themes/teams/components/Menu/menuStyles.ts]: Yes, we don't need a border for the sub-menu in default and dark theme.

File: packages/fluentui/react-northstar/src/themes/teams/components/Dropdown/dropdownVariables.ts [hunk 2]
```diff
@@ -85,9 +85,9 @@ export const dropdownVariables = (siteVars): DropdownVariables => ({
   belowListBorderRadius: `0 0 ${siteVars.borderRadiusMedium} ${siteVars.borderRadiusMedium}`,
   listBackgroundColor: siteVars.colorScheme.default.background,
   listBorderColor: 'transparent',
-  listBorderWidth: '0px',
+  listBorderWidth: '0.1rem',
```
Comment (jurokapsiar) [inline packages/fluentui/react-northstar/src/themes/teams/components/Dropdown/dropdownVariables.ts:88]: other attributes are using pxToRem, why is this one using `rem`?

Comment (vitthalr) [reply packages/fluentui/react-northstar/src/themes/teams/components/Dropdown/dropdownVariables.ts]: Thanks for pointing it out, changed it to pxToRem.

**Comment by vitthalr** [comment]
> The shadow changes seems fine but there are some weird change on the size of the dropdown such as https://screener.io/v2/states/H17FwCf88.microsoft-fluentui-fluentui/box-shadow-updates/1024x768/Chrome/dropdownexampleloadingshorthandtsx-list-with-loading-state-0 that are adding scrollbar. Just wanted to double check we want that

Thank you; `changing `rem` to `pxToRem` in `dropdownVariables.ts` solved the problem; the scrollbar is not there now.

**Comment by vitthalr** [comment]
> The shadow changes seems fine but there are some weird change on the size of the dropdown such as https://screener.io/v2/states/H17FwCf88.microsoft-fluentui-fluentui/box-shadow-updates/1024x768/Chrome/dropdownexampleloadingshorthandtsx-list-with-loading-state-0 that are adding scrollbar. Just wanted to double check we want that

@chpalac  Thank you; changing `rem` to `pxToRem` in `dropdownVariables.ts` solved the problem; the scrollbar is not there now.

**Review by ling1726 — APPROVED** [review]

**Pull request merged**